### PR TITLE
Visitor adjusts quantity in cart

### PR DESCRIPTION
**Pull request by maxglassie**
completes the functionality for adjusting an items quantity in the cart. this one was tricky because i had to use a hidden field tag to pass the item.id to the controller. 

things of note: i built the routes for cart and deleted resources :carts. all tests that use those routes still pass, but i wanted more control over what was happening while i figured out how to make the post request for the submit form. and i think this is cleaner over all.  

my total method is better than @ande1618  🥇  - who calls it sub total anyway?

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -15,8 +15,10 @@
     resources :items, except: [:index]
   end
 
-  resources :carts
+  get '/carts', to: 'carts#index'
```
Comment (alfosco) [inline config/routes.rb]: condense the cart paths with route helper

File: spec/features/cart/visitor_can_adjust_item_quantity_in_cart_spec.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          expect(page).to have_content(2)
+        end
+
+    #And the subtotal for that item should increase
+        within(".cart_item:nth-child(1)") do
+          expect(page).to have_content(2)
+        end
+
+    #And the total for the cart should match that increase
+        expect(page).to have_content("Total")
```
Comment (alfosco) [inline spec/features/cart/visitor_can_adjust_item_quantity_in_cart_spec.rb]: 'Total' is missing an integer after it.

**Comment by alfosco** [comment]
Thank you Max!

**Comment by ande1618** [comment]
@maxglassie better late than never

**Pull request merged**